### PR TITLE
pkp/pkp-lib#412 possibility to define the size of cover thumbnails

### PR DESCRIPTION
**Pull request by bozana**
I moved the code to master and separated the system specific code... :-)
The other pull request can be removed... :-)

### CODE REVIEW

File: templates/controllers/tab/settings/appearance/form/appearanceForm.tpl [hunk 1]
```diff
@@ -114,6 +114,8 @@
 	{/fbvFormArea}
 	{* end Layout *}
 
+	{$thumbnailsSizeSettings}
```
Comment (asmecher) [inline templates/controllers/tab/settings/appearance/form/appearanceForm.tpl]: Calling the variables this way is still OMP-specific. Better to use a generic name that could be used for different purposes, including (but no limited to) the thumbnail stuff in OMP.

Comment (bozana) [inline templates/controllers/tab/settings/appearance/form/appearanceForm.tpl]: Ah, me crazy... :-\
Would the name 'additionalAppereanceSettings' be OK?

Comment (asmecher) [inline templates/controllers/tab/settings/appearance/form/appearanceForm.tpl]: Yes, that sounds fine.

**Comment by bozana** [comment]
Alec, I've just seen that the changes for OMP are merged, but this should be merged too, in order for everything to function :-)
Should I in such cases also commit the submodule reference in OMP/OJS?

**Comment by asmecher** [comment]
Sorry, I missed this one. Merged now.

Generally speaking, yes, it's a good idea to commit the submodule hash too -- and see the PKP submodule changes section here: http://pkp.sfu.ca/wiki/index.php?title=Information_for_Developers
If you follow that convention for the commit message for the submodule update, then travis testing will know how to pull the right lib/pkp submodule from your fork.

**Comment by bozana** [comment]
Great, thanks! Sorry for not having done it this time, but next time for sure :-)

**Pull request merged**